src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -451,12 +451,347 @@ The following changes must be performed:
+      <code language="groovy">
+         <![CDATA[
+println "Hello World"
+]]>
+      </code>
+    </script>
+  </scriptExecutable>
+</task>
+```
+
+==== Native Scheduler Node Life Cycle
+
+As soon as tasks containing the `NS` generic information are pending, the target Native Scheduler node source will try to deploy ProActive Nodes to execute them.
+
+The node will first appear in `Deploying` <<_node_states,*state*>>. If some error occurs prior to the `nsSubmitCommand` execution (SSH connection, command syntax), the node state will change to `Lost`, with some explanation about the failure displayed.
+
+If the node remains in `Deploying` state, it is possible to monitor the job execution on the native scheduler itself (by logging into the head node):
+
+----
+root@osboxes:/tmp/activeeon_enterprise-node-linux-x64-8.1.0-SNAPSHOT/logs# qstat
+
+Job ID                    Name             User            Time Use S Queue
+------------------------- ---------------- --------------- -------- - -----
+241.osboxes                STDIN            osboxes                0 R batch          
+
+root@osboxes:/tmp/activeeon_enterprise-node-linux-x64-8.1.0-SNAPSHOT/logs# qstat -f 241
+
+Job Id: 241.osboxes
+    Job_Name = STDIN
+    Job_Owner = osboxes@osboxes
+    job_state = R
+    queue = batch
+    server = osboxes
+    Checkpoint = u
+    ctime = Fri May  4 10:13:06 2018
+    Error_Path = osboxes:/tmp/activeeon_enterprise-node-linux-x64-8.1.0-SNAPSH
+	OT/logs/Node-osboxes_852t0.out
+    exec_host = osboxes/0
+    Hold_Types = n
+    Join_Path = oe
+    Keep_Files = n
+    Mail_Points = a
+    mtime = Fri May  4 10:13:06 2018
+    Output_Path = osboxes:/tmp/activeeon_enterprise-node-linux-x64-8.1.0-SNAPS
+	HOT/logs/Node-osboxes_852t0.out
+    Priority = 0
+    qtime = Fri May  4 10:13:06 2018
+    Rerunable = True
+    Resource_List.walltime = 01:00:00
+    Resource_List.nodes = 1
+    Resource_List.nodect = 1
+    Resource_List.neednodes = 1
+    session_id = 6486
+    substate = 42
+    Variable_List = PBS_O_QUEUE=batch,PBS_O_HOME=/home/osboxes,
+	PBS_O_LOGNAME=osboxes,
+	PBS_O_PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/b
+	in:/usr/games:/usr/local/games,PBS_O_MAIL=/var/mail/osboxes,
+	PBS_O_SHELL=/bin/bash,PBS_O_LANG=fr_FR.UTF-8,
+	PBS_O_WORKDIR=/home/osboxes,PBS_O_HOST=osboxes,PBS_O_SERVER=osboxes
+    euser = osboxes
+    egroup = osboxes
+    hashname = 241.osboxes
+    queue_rank = 237
+    queue_type = E
+    comment = Job started on Fri May 04 at 10:13
+    etime = Fri May  4 10:13:06 2018
+    submit_args = -o /tmp/activeeon_enterprise-node-linux-x64-8.1.0-SNAPSHOT/l
+	ogs/Node-osboxes_852t0.out -e /tmp/activeeon_enterprise-node-linux-x64
+	-8.1.0-SNAPSHOT/logs/Node-osboxes_852t0.out -j oe
+    start_time = Fri May  4 10:13:06 2018
+    Walltime.Remaining = 3587
+    start_count = 1
+    fault_tolerant = False
+    job_radix = 0
+    submit_host = osboxes
+    init_work_dir = /home/osboxes
+    request_version = 1
+----
+
```

Comment:
The following should be part of an additional "Troubleshooting" section.

Reply:
%&!@ç !!!

Ok you're right.